R/msaGaugeRRnonrep.R

```diff
@@ -362,12 +362,11 @@ msaGaugeRRnonrep <- function(jaspResults, dataset, options, ...) {
 }
 
 .reshapeToWide <- function(dataset, measurements, parts, operators) {
-  nreplicates       <- as.vector(table(dataset[parts])[1])
-  nparts <- length(unique(dataset[[parts]]))
   dataset <- dataset[order(dataset[parts]),]
-  dataset <- cbind(dataset, data.frame(index = rep(1:nreplicates, nparts)))
+  index <- sequence(plyr::count(dataset, vars = c(parts, operators))$freq)
```

Comment:
Can this use `dplyr::count` instead? `plyr` is retired so it's probably better to use `dplyr` or another package instead.